### ISSUE TITLE
[BUG] I cannot type into the chat field with a screen reader on Android

### ISSUE THREAD

**Issue by vojtapolasek** [cogwheel0/conduit#201]
### App Version

2.2.8-71

### Platform

- [x] Android
- [ ] iOS

### How you installed it

Google Play

### What happened?

Whenever there is a text field, I see actually two of them with my screen reader (Jieshuo screen reader).
And when I try to type into it, it fails. This holds for the main chat box when I want to chat with the chosen model.
I tried to use Samsung keyboard, but it did not work.
When I open the chat, I see two edit fields:
the first is labelled "Message input, type your message" and then there is another one called "Ask conduit".
Can you please investigate this?

### What should have happened?

I expect that I will be able to use regular input method to input the text.

### How to reproduce

1. Open a new chat
2. With screen reader or some other app which shows accessibility info, inspect the screen and find two text fields
3. With screen reader on, try to type into text fields.

### Does this happen every time?

Yes, every time

### When did this start?

Since I installed the app.

**Comment by cogwheel0** [comment]
Could you try the latest release and check if it's fixed? @vojtapolasek

**Comment by vojtapolasek** [comment]
Hello, I am running 2.2.11-74 and the issue was solved only partially.
It is true that now there is only one edit box. But I still cannot type into it, the behavior stays the same.
Can I somehow help you to debug this?

**Comment by cogwheel0** [comment]
> Hello, I am running 2.2.11-74 and the issue was solved only partially. It is true that now there is only one edit box. But I still cannot type into it, the behavior stays the same. Can I somehow help you to debug this?

Thank you for the feedback. I think I was able to fix it, just tried it on my Android with the default screensreader on. Pushing it on the next patch release.

**Comment by vojtapolasek** [comment]
Hello,
sorry for not responding earlier. But this is unfortunately solved only partially, at least for my case.
ACtualy, I am (rather than Samsung Keyboard) using Advanced Braille keyboard. It is a special software keyboard which allows me to type in braille.
It is here: https://play.google.com/store/apps/details?id=abk.keyboard&hl=en
What happens with Conduit is that:
1. I click the edit box
2. Keyboard opens for me
3. I type something and confirm
4. immeditally another instance of the keyboard opens (like there would appear another edit box)
5. I type something and confirm
6. nothing is inserted in the edit box in the end.
I understand this is harder to debug, but could you please investigate?
Also I am a developer, although not Android one. So if you send me some draft APKs, I can test it out privately.
Thank you.

**Comment by cogwheel0** [comment]
> Hello, sorry for not responding earlier. But this is unfortunately solved only partially, at least for my case. ACtualy, I am (rather than Samsung Keyboard) using Advanced Braille keyboard. It is a special software keyboard which allows me to type in braille. It is here: https://play.google.com/store/apps/details?id=abk.keyboard&hl=en What happens with Conduit is that:
> 
> 1. I click the edit box
> 2. Keyboard opens for me
> 3. I type something and confirm
> 4. immeditally another instance of the keyboard opens (like there would appear another edit box)
> 5. I type something and confirm
> 6. nothing is inserted in the edit box in the end.
>    I understand this is harder to debug, but could you please investigate?
>    Also I am a developer, although not Android one. So if you send me some draft APKs, I can test it out privately.
>    Thank you.

No worries! Could you try the latest release on the play store, if that helped?

**Comment by vojtapolasek** [comment]
I am trying 2.3.9-86 and it does not work unfortunately.

**Comment by cogwheel0** [comment]
> I am trying 2.3.9-86 and it does not work unfortunately.

I couldn't try the braille app myself but could you try the new release? 2.4.1

**Comment by vojtapolasek** [comment]
Hello, it still does not work, however, I see some change.
When I click on the text field, the keyboard opens. When I do not type anything and just dismiss the keyboard, nothing else happens.
When I type something and I confirm, then immediatelly after this the keyboard opens again, as if I clicked on some field again.
Unforunately, even after typing something in the "second" instance of the keyboard and confirming, nothing appears in the text field.
